packages/avatars/src/styled/StyledAvatar.ts

```diff
@@ -80,11 +80,12 @@ const colorStyles = ({
   }
 
   if ($status) {
-    surfaceColor = $surfaceColor?.includes('.')
-      ? getColor({ variable: $surfaceColor, theme })
-      : $surfaceColor || getColor({ variable: 'background.default', theme });
+    surfaceColor =
+      $surfaceColor && /^\w+\.\w+$/u.test($surfaceColor)
```

Comment:
Is there a reason we moved away from the simpler `includes('.')` conditional here?

Reply:
Ooops, I forgot to clean up after myself. 🤦 This is what I wanted to bring up yesterday.

Will any of the [components](https://github.com/zendeskgarden/react-components/pull/1885) accepting a variable key as a user-defined color ever need to support `rgba` values created by using `getColor` with a `transparency` option?

Reply:
Ah dang, that's the sneaky notation I hadn't thought of. Let's defer to address more holistically. (I recall @geotrev suggesting some kind if `isColorVariable` utility added to `theming`).